sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -76,12 +84,20 @@ Input     : [key#x, val#x]
      
 (5) HashAggregate [codegen id : 1]
 Input: [key#x, val#x]
+Keys: [key#x]
+Functions: [partial_max(val#x)]
+FuncBufferAttrs: [max#x]
+Results: [key#x, max#x]
      
 (6) Exchange 
 Input: [key#x, max#x]
      
 (7) HashAggregate [codegen id : 2]
 Input: [key#x, max#x]
+Keys: [key#x]
+Functions: [max(val#x)]
+FuncBufferAttrs: [max#x]
+Results: [key#x, max(val#x)#x AS max(val)#x]
```

Comment:
Sorry I was wrong. I checked the source code again. The result expressions depend on `HashAggregate.aggregateAttributes`. For partial aggregate, it's the buffer attributes, but for final aggregate, it's not.

Let's show `Aggregate Attributes` instead of `FuncBufferAttrs`.

Reply:
Never mind, I just learnt a lot from it :-) 
Replaced with `Aggregate Attributes` in 9aecfaff68aeaad4b17da2fe9e640d4c76d177a8. I leave `HashAggregateExec.aggregateBufferAttributes` at the original place in this PR, although it can be abstract to base class.